packages/web/src/common/store/cache/collections/createPlaylistSaga.ts

```diff
@@ -223,7 +228,24 @@ function* createAndConfirmPlaylist(
 
     // Merge the confirmed playlist with the optimistic playlist, preferring
     // optimistic data in case other unconfirmed edits have been made.
-    const [confirmedPlaylist] = yield* call(getPlaylists, userId, [playlistId])
+    // const [confirmedPlaylist] = yield* call(getPlaylists, userId, [playlistId])
+    const { data: playlist } = yield* call(
+      [sdk.full.playlists, sdk.full.playlists.getPlaylist],
+      {
+        userId: OptionalId.parse(userId),
+        playlistId: Id.parse(playlistId)
```

Comment:
Every time I see these now, I wish I had invested some time into making SDK able to just auto-parse these things.